PlexPy.py

```diff
@@ -188,7 +188,16 @@ def main():
         'http_username': plexpy.CONFIG.HTTP_USERNAME,
         'http_password': plexpy.CONFIG.HTTP_PASSWORD,
     }
-    webstart.initialize(web_config)
+    web_start = False
+    while not web_start and web_config['http_port'] < http_port + 5:
+        web_start = webstart.initialize(web_config)
+        web_config['http_port'] += 1
+
+    if web_start:
+        http_port = web_config['http_port']
+        plexpy.CONFIG.HTTP_PORT = http_port
```

Comment:
If you do go through with this feature, you should log a (warn, error?) here so the user can check the log to see the port it's running on.